docs/book.toml

```diff
@@ -5,8 +5,9 @@ language = "en"
 multilingual = false
 src = "src"
 
-[output.linkcheck]
-follow-web-links = false
-traverse-parent-directories = false
-warning-policy = "warn"
-exclude = [ 'README.md' ]
+# FIX: the link check prevents some pages to be generated
```

Comment:
This whole section was removed for your local testing only or you want it this way in the repo as well? If that's the case what's the impact?

Reply:
Without link checks the risk is high to have dandling links. Unfortunately, I noticed that something is wrong preventing the generation of some pages without any error messages. Hence, I prefer to remove it for now. That will have to be fixed properly later.